packages/react-native/src/__tests__/index.spec.ts

```diff
@@ -0,0 +1,8 @@
+import * as exported from '..';
+
+describe('@aws-amplify/ui-react-native', () => {
+  it('exports should match snapshot', () => {
+    const exportedKeys = Object.keys(exported);
```

Comment:
Are keys guaranteed to be in order?

Reply:
They should ordered be according to order of assignment, which should change unless we change the exports order. Cam change if you feel strongly else will update if needed in the future